dao/src/main/java/se/fortnox/reactivewizard/db/query/parts/CollectionOptionsQueryPart.java

```diff
@@ -36,11 +41,33 @@ public void visit(StringBuilder sql, Object[] args) {
             if (collectionOptions != null) {
                 if (collectionOptions.getSortBy() != null) {
                     String sortBy = CamelSnakeConverter.camelToSnake(collectionOptions.getSortBy());
-                    for (String allowed : queryAnnotation.allowedSortColumns()) {
-                        if (allowed.equals(sortBy)) {
-                            collectionOptionsHasOrderBy = true;
-                            addOrderBy(sql, buildOrderBy(collectionOptions.getOrder(), allowed));
-                            break;
+                    var sortByParts = sortBy.split(",");
```

Comment:
Could be good to have a check for max length for the query param unless Netty's query decoder already checks that?

Reply:
I do not know. sortBy is however not a new parameter and that change request is not addressing this change.

Reply:
Sure, but we are now splitting it and running a regex for each column when sorting by multiple columns compared to only sorting by a single column with an equals comparison? I'd say 256 bytes is a reasonable limit, but we can put it at 1024 to be on the safe side, doesn't make that much of a difference performance wise.